openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -55,15 +56,16 @@ <h4 class="question__title ui-toggle-visibility__control">
             </div>
         </li>
         {% endfor %}
-        <li class="wrapper--input field field--textarea assessment__rubric__question assessment__rubric__question--feedback" id="assessment__rubric__question--feedback">
-            <label class="question__title" for="assessment__rubric__question--feedback__value">
-                <span class="question__title__copy">{{ rubric_feedback_prompt }}</span>
+        <li class="wrapper--input field field--textarea assessment__rubric__question assessment__rubric__question--feedback">
+            <label class="question__title" for="{{ rubric_type }}__assessment__rubric__question--feedback__value">
+                <span class="question__title__copy">{% trans rubric_feedback_prompt %}</span>
```

Comment:
Do we know that this will send the right string to `gettext` for translation (i.e. are there other places we do it in ora2)? I'm not super familiar with how it works for ora2.